dotnetv3/cross-service/AuroraItemTracker/Readme.md

```diff
@@ -0,0 +1,180 @@
+
+### Aurora Item Tracker user interface
+To start the React web application, you can download files from the following GitHub repository. Included in this repository are instructions on how to set up the project. Click the following link to access the GitHub location [Item Tracker web client](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/resources/clients/react/item-tracker/README.md).  
+
+When the web application is running, you will see something like the following.
+
+![Aurora Item Tracker UI](images/elapp1.png)
+
+#### Use the React web application
+
+A user can perform these tasks using the web application:
+
+1. View all items
+1. View active items only
+1. View archived items only
+1. Add a new item
+1. Change an active item into an archived item.
+1. Send a report as an email attachment
+
+The web application displays *active*, *archived*, or all items. For example, the following illustration shows the React application displaying active data.
+
+![Viewing active items](images/elapp2.png)
+
+Here is the React application displaying archived data.
+
+![Viewing archived items](images/elapp3.png)
+
+The user can insert a new item into the **items** table from the user interface in the React front end application. In the following image, you can see the new item form. 
+
+![Adding a new work item](images/item_tracker_add_item.png)
+
+The user can enter an email recipient in the **Email** text field and choose **Send Report**. The recipient will get an email something like the following.
+
+![Report email](images/email.png)
+
+⚠️ **Important!** You must update the email **sender** address with a verified email address. Otherwise, the email is not sent. For more information, see [Verifying email addresses in Amazon SES](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/verify-email-addresses.html).       
+
+Active items are selected from the database and used to dynamically create a .csv document. Then, the application uses **Amazon SES** to email the document to the email address entered. The following image shows an example of a report.
+
+![A sample spreadsheet](images/excel_spreadsheet.png)
+
+## Delete the resources
+
+To avoid charges, delete all the resources that you created for this tutorial.
+
+If you created the resources using the CDK script provided, you can delete the resources by running the script with the following command:
+
+```
+cdk destroy
+```
+
+If you created your resources through the AWS Management Console, or modified them by running the app, you must use the console to delete them.
+
+**Note**: By running the app, you modified the table, so you must delete these resources manually through the console before you can delete the stack.
+
+## Next steps
+
+Congratulations! You have created and run a RESTful C# API that manages data in an Amazon Aurora serverless database, explored the database through the React frontend web application, and sent reports using Amazon SES.
```

Comment:
I would use the same terminology you use in the intro paragraph: "use the AWS SDK for .NET (v3) to create a REST service" instead of "RESTful C# API".

Reply:
Fair point. Updated.